packages/workflows/src/handlers/inventory/restore-inventory-items.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  IInventoryService,
+  InventoryItemDTO,
+  SharedContext,
+} from "@medusajs/types"
+import { WorkflowArguments } from "../../helper"
+
+export async function restoreInventoryItems({
+  container,
+  context,
+  data,
+}: WorkflowArguments<{
+  inventoryItems: { inventoryItem: InventoryItemDTO }[]
+}>) {
+  const { manager } = context as SharedContext
+  const inventoryService: IInventoryService =
+    container.resolve("inventoryService")
```

Comment:
q: Wouldn't this throw an error if its not able to resolve it?

Reply:
It will be undefined. It should throw if not found but I guess we register `undefined` under that key during bootstrap?